doc/dev/installation.rst

```diff
@@ -203,7 +203,7 @@ You can also choose to build the custom Catalyst dialects only, with:
 Frontend
 """"""""
 
-To install the ``pennylane-catalyst`` Python package (the compiler frontend) in editable mode:
+To install the ``PennyLane-Catalyst`` Python package (the compiler frontend) in editable mode:
```

Comment:
same here, I would prefer to leave this lowercase

Reply:
Reverted